nncf/ptq/openvino/engine.py

```diff
@@ -0,0 +1,144 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Optional
+from typing import Callable
+
+from openvino.tools import pot
+
+from nncf.ptq.api.types import ModelType
+from nncf.ptq.api.dataloader import NNCFDataLoader
+from nncf.ptq.data.utils import create_subset
+
+
+class CustomMetric(pot.Metric):
```

Comment:
I didn't get why we need this class. Can you explain?

Reply:
It was discussed offline. I renamed this class: `CustomMetric` -> `DummyMetric`